codebuild/Dockerfile.cpu

```diff
@@ -20,7 +20,7 @@
   RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
   RUN update-alternatives --install /usr/local/bin/pip pip /usr/local/bin/pip3 1
   
-  RUN pip install torch torchvision -f https://download.pytorch.org/whl/torch_stable.html
+  RUN pip install torch torchvision torchtext -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
curious, why is this change here?

Reply:
As a part of this PR we added changes in the sanity script for validating the object_detector default handler. Later we thought of adding the changes to validate the remaining default handlers as well. For validating the text_classification default handler we needed torchtext installed in the codebuild docker image.

Reply:
@mycpuorg 
Coming to think of it, the new regression suite also validates all the default handlers. This looks like a repetition of work and managing the same regression in two different places.